src/schema/v2/partner/CommerceOptIn/__tests__/commerceOptInMutation.test.ts

```diff
@@ -0,0 +1,54 @@
+import gql from "lib/gql"
+import { runAuthenticatedQuery } from "schema/v2/test/utils"
+
+describe("CommerceOptInMutation", () => {
+  const mutation = gql`
+    mutation {
+      CommerceOptIn() {
+        CommerceOptInOrError {
+          __typename
+          ... on CommerceOptInSuccess {
+            // 
+          }
+          ... on CommerceOptInFailure {
+            mutationError {
+              message
+            }
+          }
+        }
+      }
+    }
+  `
```

Comment:
I'd say you want something here that will follow the same structure you'd use for the real mutation. It would look something like:
```graphql
mutation
  { 
    commerceOptInMutation(input: { id: <partner-id>, ...}) {
      commerceOptInMutationOrError {
        __typename
        ... on CommerceOptInSuccess {
          updatedCommerceOptIn {
            ids
          }
          skippedCommerceOptIn {
            ids
          }
        }
        ... on CommerceOptInFailure {
          mutationError {
            error
            message
          }
        }
      }
    }
}
```